jpa/eclipselink.jpa.testapps/jpa.test.ddlgeneration/src/main/java/org/eclipse/persistence/testing/models/jpa/ddlgeneration/Event.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ * Copyright (c) 2025 IBM Corporation. All rights reserved.
```

Comment:
Are You IBM employee?

Reply:
No, but this code is taken from https://github.com/eclipse-ee4j/eclipselink/issues/2451 - where IBM employee provided this piece of code.